ui/app/clinical/common/views/providerInfo.html

```diff
@@ -0,0 +1,12 @@
+<div class="provider-info">
+    <table>
+        <tr>
+            <td>Doctor Name: <span>{{provider[0].name}}</span>
```

Comment:
Does this mean -- pick the first provider name for printing?

Reply:
yeah. Provider is itselft a list for each encounter